apis/cluster.cattle.io/v1/types.go

```diff
@@ -219,6 +221,8 @@ type ClusterNode struct {
 	v1.Node
 	NodeName    string
 	ClusterName string
+	Requested   v1.ResourceList `json:"requested,omitempty"`
```

Comment:
Would it be possible to make it a part of the NodeSpec? Along with the Allocatable/Capacity